### PR TITLE
#4731 [Manager] Prevent errors from swallowing by epics

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/eventLog/createEventLogEpic.js [hunk 1]
```diff
@@ -20,21 +20,30 @@ import { concatMap, map, catchError } from 'rxjs/operators';
...
 });
 
 const setLogToLocalForge$ = (values) => {
+  let { err } = values;
+  err = isError(err)
+    ? {
+        title: err.message,
+        data: { error: { message: err.stack } },
+      }
+    : err;
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/eventLog/createEventLogEpic.js:37]: Looks okay to me. @cc @eechih @saivirtue

Comment (eechih) [inline ohara-manager/client/src/store/epics/eventLog/createEventLogEpic.js:37]: 覺得這段邏輯放在 Epic 的 catchError 比較適合 🤔
或許我們應該可以客製一個 `catchError` operator，用來轉換 native error object。

**Comment by wu87988622** [comment]
retry

**Comment by wu87988622** [comment]
retry

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**